src/js/actions/tools.js

```diff
@@ -152,7 +153,8 @@ define(function (require, exports) {
             // The resize rectangles are roughly 8 points radius
             inset = 4,
             // In case of artboards, we have no rotate, so we can stay within the border
-            outset = artboards ? inset : 27;
+            outset = artboards ? inset : 27,
+            distortModifier = system.isMac ? { command: true } : { control: true };
```

Comment:
seems like this pattern gets repeated a few times throughout DS, so in the **future** we may consider abstracting to ~ `getSuper()`